rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -539,6 +539,42 @@ int wait4child(pal_exec_args *attr)
 	return 0;
 }
 
+int __pal_get_local_report(void *targetinfo, int targetinfo_len, void *report, int* report_len)
+{
+	uint8_t report_data[64] = { 0, };
+	struct sgx_report report_align;
+	int ret;
+
+	if (!initialized) {
+		fprintf(stderr, "Enclave runtime skeleton uninitialized yet!\n");
```

Comment:
Use stdio->stderr from the parameter of __pal_create_process instead of using the standard error. You have context and opportunity to record stdio->stderr.